src/command.js

```diff
@@ -144,6 +144,11 @@ Command.prototype._prepare = function(options) {
     logger.transports.console.level = "none";
   }
 
+  var configPath = utils.getInheritedOption(options, "configPath");
+  if (configPath) {
```

Comment:
is this check necessary? if it never finds a value, it returns undefined, which is the same as not setting the value (below)

Reply:
You are right, unnecessary check. Removed it